providers/class.two-factor-email.php

```diff
@@ -152,6 +170,9 @@ public function authentication_page( $user ) {
 			<label for="authcode"><?php esc_html_e( 'Verification Code:', 'two-factor' ); ?></label>
 			<input type="tel" name="two-factor-email-code" id="authcode" class="input" value="" size="20" pattern="[0-9]*" />
 		</p>
+		<p class="two-factor-email-resend">
+			<input type="submit" class="button" name="two-factor-email-code-resend" value="<?php esc_attr_e( 'Resend Code', 'two-factor' ); ?>" />
```

Comment:
I would prefer to have this as a regular link and a GET request instead but we don't have access to the current 2FA request token.